tests/test_jinja_macros.py

```diff
@@ -119,7 +119,7 @@ def test_criteria_macro_commenter(self, request_ctx):
             full_name="Test Commenter",
             email="test@example.com",
             roles=["COF_COMMENTER"],
-            highest_role="COF_COMMENTER",
+            highest_role_map="COF_COMMENTER",
```

Comment:
This should maybe be a map, but it's not used anymore so might be able to remove the g.user from this test all together?

Reply:
User class has an attribute "highest_role_map" in utils?

Reply:
I would rather have a conversation with the person who has been working on this functionality but my focus is more on the functionality I have been working on related to mapping and formatting.